src/shared/DotnetTool/pack-tool.sh

```diff
@@ -91,49 +79,54 @@ fi
 # Publish core application executables
 echo "Publishing core application..."
 $DOTNET_ROOT/dotnet publish "$GCM_SRC" \
-	--configuration="$CONFIGURATION" \
-	--framework="$FRAMEWORK" \
-	--output="$(make_absolute "$PAYLOAD")" \
+    --configuration="$CONFIGURATION" \
+    --framework="$FRAMEWORK" \
+    --output="$(make_absolute "$PAYLOAD")" \
     -p:UseAppHost=false || exit 1
 
 echo "Publishing Bitbucket UI helper..."
 $DOTNET_ROOT/dotnet publish "$BITBUCKET_UI_SRC" \
-	--configuration="$CONFIGURATION" \
-	--framework="$FRAMEWORK" \
-	--output="$(make_absolute "$PAYLOAD")" \
+    --configuration="$CONFIGURATION" \
+    --framework="$FRAMEWORK" \
+    --output="$(make_absolute "$PAYLOAD")" \
     -p:UseAppHost=false || exit 1
 
 echo "Publishing GitHub UI helper..."
 $DOTNET_ROOT/dotnet publish "$GITHUB_UI_SRC" \
-	--configuration="$CONFIGURATION" \
-	--framework="$FRAMEWORK" \
-	--output="$(make_absolute "$PAYLOAD")" \
+    --configuration="$CONFIGURATION" \
+    --framework="$FRAMEWORK" \
+    --output="$(make_absolute "$PAYLOAD")" \
     -p:UseAppHost=false || exit 1
 
 echo "Publishing GitLab UI helper..."
 $DOTNET_ROOT/dotnet publish "$GITLAB_UI_SRC" \
-	--configuration="$CONFIGURATION" \
-	--framework="$FRAMEWORK" \
-	--output="$(make_absolute "$PAYLOAD")" \
+    --configuration="$CONFIGURATION" \
+    --framework="$FRAMEWORK" \
+    --output="$(make_absolute "$PAYLOAD")" \
     -p:UseAppHost=false || exit 1
 
 # Collect symbols
 echo "Collecting managed symbols..."
 mv "$PAYLOAD"/*.pdb "$SYMBOLOUT" || exit 1
 
+# Copy DotnetToolSettings.xml file
+echo "Copying out package configuration files..."
+cp "$SRC/$DOTNET_TOOL/DotnetToolSettings.xml" "$PAYLOAD/"
+
+# Copy package icon image
+echo "Copying images..."
+cp "$SRC/$DOTNET_TOOL/icon.png" "$IMGOUT" || exit 1
+
 echo "Build complete."
 
 #####################################################################
 # Pack dotnet tool
 #####################################################################
 echo "Creating dotnet tool package..."
 
-mkdir -p "$PACKAGE" || exit 1
-echo "Laying out files..."
-cp -r "$SRC/$DOTNET_TOOL/DotnetToolSettings.xml" \
-    "$PAYLOAD/." \
-    "$PACKAGE/"
```

Comment:
No need to copy the output to another directory just to pack it. Instead we're now `dotnet publish`-ing outputs to a payload directory, and copying associated .xml/.png files to the right dir structure in the first place.